### PR TITLE
Add optional arg timeout to command pan-os-register-user-tag

### PR DESCRIPTION
**Pull request by ali-sawyer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
n/a

## Description
Add optional argument `timeout` to command `pan-os-register-user-tag` so user can optionally specify amount of time after which to automatically unregister tag from user. This same option already exists for IPs in the similar command `pan-os-register-ip-tag`. Also update integration's docker container to latest version.

## Screenshots
n/a

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ali-sawyer** [comment]
@GuyAfik FYI

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -3922,8 +3925,13 @@ def panorama_register_user_tag_command(args: dict):
         raise Exception('The panorama-register-user-tag command is only available for PAN-OS 9.X and above versions.')
     tag = args['tag']
     users = argToList(args['Users'])
+    # if not given, timeout will be 0 (never expires)
+    timeout = int(args.get('timeout', '0'))
+
+    if major_version <= 8 and timeout:
+        raise DemistoException('The timeout argument is only applicable on 9.x PAN-OS versions or higher.')
```
Comment (GuyAfik) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:3932]: This entire command is only supported on 9.x, see the code in line 3924, hence this check is redundent

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 2]
```diff
@@ -5391,6 +5393,8 @@ script:
       isArray: true
       name: Users
       required: true
+    - description: Timeout value to automatically unregister the users (in seconds). Only applicable to PAN-OS 9.x and higher.
+      name: timeout
```
Comment (GuyAfik) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:5397]:
```suggestion
    - description: Timeout value to automatically unregister the users (in seconds). Only applicable to PAN-OS 9.x and higher.
      name: timeout
      required: false
```

File: Packs/PAN-OS/ReleaseNotes/1_9_1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Add optional argument `timeout` to command `pan-os-register-user-tag`.
```
Comment (GuyAfik) [inline Packs/PAN-OS/ReleaseNotes/1_9_1.md:4]:
```suggestion
- Added the argument *timeout* to command ***pan-os-register-user-tag***.
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 4]
```diff
@@ -1954,6 +1954,8 @@ script:
...
```
Comment (GuyAfik) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:1958]: The ```panorama-register-user-tag``` is deprecated (see line 1961). Therefore we want to encourage our customers to use the ```pan-os-register-user-tag``` instead.

You simply remove it and implement it only on ```pan-os-register-user-tag``` command.

File: Packs/PAN-OS/ReleaseNotes/1_9_1.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+- Update the Docker image to: *demisto/python3:3.10.1.27636*.
```
Comment (GuyAfik) [inline Packs/PAN-OS/ReleaseNotes/1_9_1.md:5]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.1.27636*.
```

**Comment by GuyAfik** [comment]
Hi @ali-sawyer, as usual you are doing an incredible work! keep up with that! Your contributions are much appreciated! I gave some comments, feel free to reach me our if something isn't clear

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py [hunk 6]
```diff
@@ -3922,8 +3925,13 @@ def panorama_register_user_tag_command(args: dict):
         raise Exception('The panorama-register-user-tag command is only available for PAN-OS 9.X and above versions.')
     tag = args['tag']
     users = argToList(args['Users'])
+    # if not given, timeout will be 0 (never expires)
+    timeout = int(args.get('timeout', '0'))
```
Comment (GuyAfik) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.py:3929]:
```suggestion
    timeout = arg_to_number(args.get('timeout', '0'))
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 1 above)
Comment (ali-sawyer) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:3932]: Agreed, removing.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml (hunk 4 above)
Comment (ali-sawyer) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:1958]: Agreed, removing.

File: Packs/PAN-OS/Integrations/Panorama/Panorama.py (hunk 6 above)
Comment (ali-sawyer) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.py:3929]: Changing both here and for the same thing in the `panorama_register_ip_tag_command` function.

**Comment by GuyAfik** [comment]
Looks good, was tested in the demo. Thank you @ali-sawyer for your contribution!

**Pull request merged**